README.md

```diff
@@ -7,56 +7,190 @@ Lisk is a next generation crypto-currency and decentralized application platform
+        brew install curl automake autoconf libtool
+        ```
+
+        _(Recommended)_ Install Lunchy for easier starting and stoping of services:
+
+        ```
+        gem install lunchy
+        ```
 
 - Git (<https://github.com/git/git>) -- Used for cloning and updating Lisk
 
-  `sudo apt-get install -y git`
+    - Linux:
 
-- Node.js (<https://nodejs.org/>) -- Node.js serves as the underlying engine for code execution.
+        ```
+        sudo apt-get install -y git
+        ```
 
-  System wide via package manager:
+    - Mac:
 
-  ```
-  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
-  sudo apt-get install -y nodejs
-  ```
+        ```
+        brew install git
+        ```
 
-  Locally using [nvm](https://github.com/creationix/nvm):
+### Node.js (<https://nodejs.org/>)
 
-  ```
-  curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
-  nvm install v6.10.1
-  ```
+- Node.js serves as the underlying engine for code execution.
+
+    Install System wide via package manager:
+
+    - Linux:
+
+        ```
+        curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+        sudo apt-get install -y nodejs
+        ```
+
+    - Mac:
+
+        ```
+        brew install node@6
+        ```
+
+- _(Recommended)_ Install n -- Used for Node.js version management
 
-- Install PostgreSQL (version 9.6.2):
+    ```
+    npm install -g n
+    n 6.12.1
+    ```
+
+- _(Recommended)_ PM2 (<https://github.com/Unitech/pm2>) -- PM2 manages the node process for Lisk
 
   ```
-  curl -sL "https://downloads.lisk.io/scripts/setup_postgresql.Linux" | bash -
-  sudo -u postgres createuser --createdb $USER
-  createdb lisk_test
-  createdb lisk_main
-  sudo -u postgres psql -d lisk_test -c "alter user "$USER" with password 'password';"
-  sudo -u postgres psql -d lisk_main -c "alter user "$USER" with password 'password';"
+  npm install -g pm2
   ```
 
-- Bower (<http://bower.io/>) -- Bower helps to install required JavaScript dependencies.
+#### Special note about NPM 5
+
+Due to an issue with NPM 5.4.x and higher, node-sodium currently cannot be built. Therefore it is recommended to fixate the local NPM version at 5.3.x if you are running NPM 5.4.x or higher.
+
+All Systems - This may require sudo depending on your environment:
+
+```
+npm install -g npm@5.3.0
+```
+
+### PostgreSQL (version 9.6.6):
+
+   - Linux:
+
+        ```
+        curl -sL "https://downloads.lisk.io/scripts/setup_postgresql.Linux" | bash -
+        sudo -u postgres createuser --createdb $USER
+        createdb lisk_test
+        createdb lisk_main
+        sudo -u postgres psql -d lisk_test -c "alter user "$USER" with password 'password';"
+        sudo -u postgres psql -d lisk_main -c "alter user "$USER" with password 'password';"
+        ```
+
+   - Mac:
+
+        When installing a different version, replace 9.6.6 with your version in the following commands:
+
+        ```
+        brew install postgresql
```

Comment:
The current version of brew install a 10-th version of PostgreSQL. It seems to work fine on my local machine, but unless it is not the official supported version maybe we should advise installing 9.6 explicitly. (brew install postgresql@9.6)

Reply:
Done 👍